app/views/users/list.html.erb

```diff
@@ -19,7 +19,7 @@
       <th><%= t('users.list.last_activity') %></th>
       <th><%= t('users.list.history') %></th>
       <th><%= t('users.list.joined') %></th>
-      <% if current_user && current_user.role = "admin" && current_user.role = "moderator" %>
+      <% if current_user && current_user.role == "admin" && current_user.role == "moderator" %>
```

Comment:
good catch here!!! But I think i agree w/ the above commenter that we need to use `||` instead of `&&` for this and line 8 -- so it's moderator OR admin. Thank you!!!